src/tests/evaluation.test.ts

```
@@ -0,0 +1,128 @@
+        });
+
+      const report = await evaluationService.evaluate(testCases);
+
+      expect(report).toContain('Test execution failed');
+      expect(report).toContain('❌ Failed');
+    });
+
+    it('should calculate correct pass/fail statistics', async () => {
+      // Mock the executeTest method to return different results
+      jest
+        .spyOn(evaluationService as any, 'executeTest')
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)